### ISSUE TITLE
Allow alias commands of Plugins  to be available in Parent Repo

### ISSUE THREAD

**Issue by aronggile** [thought-machine/please#2648]
It would be nice if alias commands defined in a plugin repo could be made available 
in client repos either automatically or with minimal configuration.  This would make it 
easier to build and share tools that simplify  interaction with plugins.

For example, I am currently working on  please plugin that provides a Terraform module, 
which can be used to spin up a local QEMU/KVM test environment.  I would like to
just import this plugin into an external project I am working on, say a Go service, and execute 
aliases it provides directly. 

 Something like,
-  `plz infra-up`  to launch the infrastructure
-  `plz infra-down` to shut it down

*Where* `infra-up` and `infra-down` are aliases defined in the Terraform plugin repo.

This is better than providing additions shell script or a `make` file for interacting
 with a project that already uses `please`. I also think for small team driven or 
solo projects, where devs are expected to ware multiple hats, such a feature may 
be useful.

That said,  given Please is a do one thing and one thing only properly type of Linuxy project, this may be
outside of the project's scope. It may also present security challenges as this seems lie a gateway 
to `plz install` command.  Those of you with more experience can look into it, but I thought it was worth sharing.

**Comment by Tatskaari** [comment]
Yeah, this is something I've been musing about. I similarly want to add sub-commands to go the go plugin to generate and format targets. Once we have v17 out the door and plugins rolled out, I'd like to come back to this.
